std/typecons.d

```diff
@@ -2068,35 +2068,55 @@ if (is(T == class) || is(T == interface) || isDynamicArray!T || isAssociativeArr
 Convenience function for creating a $(D Rebindable) using automatic type
 inference.
 
+If a $(LREF Rebindable) obj is passed to `rebindable`, it is returned without modifications.
```

Comment:
IMO the documentation should not contain shortenings like this. "obj" can be removed without affecting the clarity of the text.

Reply:
This was copy/pasted from the other documentation header, but yeah good idea to fix.